modules/monitoring/library/Monitoring/Backend/Ido/Query/HoststatusQuery.php

```diff
@@ -117,35 +117,25 @@ class HoststatusQuery extends IdoQuery
             'host_process_performance_data'         => 'hs.process_performance_data',
             'host_retry_check_interval'             => 'hs.retry_check_interval',
             'host_scheduled_downtime_depth'         => 'hs.scheduled_downtime_depth',
-            'host_severity'                         => 'CASE WHEN hs.current_state = 0
-            THEN
-                CASE WHEN hs.has_been_checked = 0 OR hs.has_been_checked IS NULL
-                     THEN 16
-                     ELSE 0
-                END
-                +
-                CASE WHEN hs.problem_has_been_acknowledged = 1
-                     THEN 2
-                     ELSE
-                        CASE WHEN hs.scheduled_downtime_depth > 0
-                            THEN 1
-                            ELSE 4
-                        END
-                END
+            'host_severity'                         => 'CASE
+            WHEN hs.current_state = 0
```

Comment:
You must check the has_been_checked column first before checking anything else. The default value for current_state is 0 anyway.